src/components/HealthMarkers.tsx

```diff
@@ -0,0 +1,53 @@
+// maps over 'healthMarkers' array
+// 1. checks 'type' to render icon and heading
+// 2. renders 'MarkerList' component, which will render individual cards
+
+// had to reformat data a little bit to optimize. 'healthMarkers' is an array of objects. Each object has a 'type' key that allows us to map all 3. The individual 'markers' data within each type is another array that we can esaily map through to render data.
+
+'use client'
+import healthData from '@/app/cohort_template'
+import styles from '../app/results/page.module.css'
+import MarkerList from './MarkerList'
+import { FaHeartbeat } from 'react-icons/fa'
+import { FaDumbbell } from 'react-icons/fa6'
+import { TbStretching } from 'react-icons/tb'
+
+const HealthMarkers = () => {
+  const { healthMarkers } = healthData['female_40_59']
+
+  return (
+    <div className={styles.markersContainer}>
+      {healthMarkers.map((item, i) => {
+        return (
+          <div className={styles.listsContainer} key={i}>
+            <div className={styles.listHeader}>
+              {/* CHECKS TYPE ---------------------------- */}
+              {item.type === 'cardiovascular' ? (
+                <>
+                  <FaHeartbeat style={{ fontSize: '40px' }} />
```

Comment:
It's always good to be DRY as possible when working with UI code. If you have multi conditional logic like you do on lines 24 - 43, look for ways to reduce repetitive code.

For this component we could do the following: 
1. Create a map for the `item.type` and matching Font Awesome icon:
`const iconMappings = {'physical: <FaDumbbell stlye={{ fontSize: '40px'}} />, 'flexibility': <TbStretching style={{ fontSize: '40px' }} /> ...}`
2. Refactor lines 24 - 43 to be`<> iconMappings[item.type]<h1>{item.type}</h1></>`

This will make your code a lot more DRY